### DISCUSSION TITLE
Possible to deep link directly to the archive page

### DISCUSSION THREAD

**Discussion by sorentv2reg** [open-meteo/open-meteo#824] [Ideas]
Hi

Thanks for an absolutely wonderful system. I've extracted a lot of historical data and during testing it would be nice if you could link directly to the archive page for a specific location and time, like:

https://open-meteo.com/en/docs/historical-weather-api#latitude=57.048&longitude=9.9187&start_date=2023-06-09&end_date=2023-06-09&hourly=temperature_2m,apparent_temperature&timezone=Europe%2FBerlin

But for some reason start_date and end_date don't work (although they are injected into the URL when you use the date selector).

**Comment by patrick-zippenfenig** [comment]
Could you create an issue here: https://github.com/open-meteo/open-meteo-website? Thanks!
